db_release_scripts/README.MD

```diff
@@ -1,144 +1,145 @@
+Properties file variable details:
 
-**DB_SERVERIP=** Contains details of Destination SERVER_IP(Ex:10.0.0.1) where the deployment is targeted
+**DB_SERVERIP=** Contains details of Destination SERVER_IP(Ex:10.0.0.1) where the deployment is targeted.
 
-**DB_PORT=** Contains the postgres server port details where the postgres is allowed to connect. Ex: 5433
+**DB_PORT=** Contains the postgres server port details where the postgres is allowed to connect. Ex: 5433.
 
-**SU_USER=** Contains the postgres super user name to connect to the postgres database i.e. postgres
+**SU_USER=** Contains the postgres super user name to connect to the postgres database i.e. postgres.
 
-**DEFAULT_DB_NAME=** Default database name to connect with respective postgres server i.e. ex: postgres
+**DEFAULT_DB_NAME=** Default database name to connect with respective postgres server i.e. ex: postgres.
 
 **MOSIP_DB_NAME=** Database name for which the deployment is scheduled.
 
-**SYSADMIN_USER=** Variable contains the sysadmin use name which indeed is going to be the super user for the remaining actions going to be performed by shell script.
+**SYSADMIN_USER=** Variable contains the sysadmin user name which is super user for the remaining actions going to be performed by shell script.
 
-**BASEPATH=** Path for DB scrips which are kept in the Deployment server. Ex: /home/madmin/database_release
+**BASEPATH=** Path for DB scrips which are kept in the Deployment server. Ex: /home/madmin/database_release.
 
-**LOG_PATH=** Path where deployment log file will be created. Ex: /home/madmin/logs/
+**LOG_PATH=** Path where deployment log file will be created. Ex: /home/madmin/logs/ .
 
-**ALTER_SCRIPT_FLAG=** Flag variable which contains value as 0 or 1 for any Alter scripts existance for the particular DB. if flag=0 then no alterscripts else flag=1.
+**ALTER_SCRIPT_FLAG=** Flag variable which contains value as 0 or 1 to determine any alter scripts existance for the particular DB. If flag=0 then no alterscripts else flag=1.
 
 **ALTER_SCRIPT_FILENAME=** Alter scripts sql filename, This file is kept under sql directory, ex: <release_version_number>_<schema_name>-scripts_release.sql.
 
-**REVOKE_SCRIPT_FLAG=** Flag variable which contains value as 0 or 1 for any revoke scripts existance for the particular DB. if flag=0 then no revokescripts else flag=1.
+**REVOKE_SCRIPT_FLAG=** Flag variable which contains value as 0 or 1 to determine any revoke scripts existance for the particular DB. if flag=0 then no revoke scripts else flag=1.
 
 **REVOKE_SCRIPT_FILENAME=** Revoke scripts sql filename, This file is kept under sql directory, ex: <release_version_number>_<schema_name>-scripts_revoke.sql.
 
-**Note - Make sure, There is a single empty line at end of the .properties files content and No spaces in beggining and end of the parameter values**
+**Note - There should be a empty line at end of the .properties files and No spaces in beginning and end of the parameter values**
```

Comment:
Grammar 

**Note** - There should be an empty line at the end of the `.properties` files and no spaces in the beginning and at the end of the parameter values.